lib/ansible/galaxy/__init__.py

```diff
@@ -55,7 +55,7 @@ def __init__(self):
 
         # load data path for resource usage
         this_dir, this_filename = os.path.split(__file__)
-        type_path = context.CLIARGS.get('role_type', 'default')
+        type_path = context.CLIARGS.get('{0}_type'.format(context.CLIARGS.get('type')), 'default')
```

Comment:
Could you please use named placeholders? Numbers are rather confusing and it's usually important to care about the maintainability of such things.

```suggestion
        type_path = context.CLIARGS.get('{content_type}_type'.format(content_type=context.CLIARGS.get('type')), 'default')
```